src/components/actionJSButton.js

```diff
@@ -124,7 +119,7 @@
       return undefined;
     };
 
-    const showIndicator = isLoading;
+    const showIndicator = isLoading || loading;
```

Comment:
Why do we have two loading variables?

Reply:
isLoading can be used by the builder to set te loading state. Loading will be returned from  useActionJS when the action is running, but doesn't set the state and useActionJS doesn't have an onLoading option to set te state either.

Reply:
I think we should change the name of `loading` to `isActionLoading` to make it clear what we are waiting for!

Reply:
Changed it to isLoadingAction